### PR TITLE
add speech to text support and image support

### PR DESCRIPTION
**Pull request by alifhsgigalogy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,12 +1,13 @@
 {
-  "name": "react-chatbot-kit",
-  "version": "2.2.0",
+  "name": "react-chatbot-kit-for-gpt4-vision",
```
Comment (moinsam) [inline package.json]: Lets keep the same name as the repo name `react-chatbot-kit-extension`.
With your current name if gpt5 arrive we have to change it.

**Comment by moinsam** [comment]
- Add copyright notice (`Copyright (c) 2024 Gigalogy`) below the original Author's copyright notice.
- Update webpack to `5.76.0`

**Review by moinsam — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (alifhsgigalogy) [reply package.json]: resolved

**Comment by alifhsgigalogy** [comment]
> * Add copyright notice (`Copyright (c) 2024 Gigalogy`) below the original Author's copyright notice.
> * Update webpack to `5.76.0`

updated the Copyright and Webpack version

**Review by moinsam — APPROVED** [review]

**Pull request merged**